apps/nextjs/src/middleware.ts

```diff
@@ -1,11 +1,11 @@
 import { authMiddleware } from "@clerk/nextjs";
 
-// This example protects all routes including api/trpc routes
-// Please edit this to allow other routes to be public as needed.
 // See https://clerk.com/docs/references/nextjs/auth-middleware for more information about configuring your Middleware
-export default authMiddleware();
+export default authMiddleware({
+  publicRoutes: ["/"], // Do not protect '/' for now since that's where we land
+});
 
-// Do not protect '/' for now since that's where we land
+// See https://nextjs.org/docs/pages/building-your-application/routing/middleware#matcher for what config does
 export const config = {
-  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/(api|trpc)(.*)"],
+  matcher: ["/((?!.+\\.[\\w]+$|_next).*)", "/", "/(api|trpc)(.*)"],
```

Comment:
We don't need to protect the API routes, as we are already running our own middleware on those routes. I think that you can just remove it from the matcher

Reply:
We should add it to the public routes then, since if we remove it altogether no TRPC request is able to authenticate due to the following error:

```
❌ tRPC failed on <no-path>: Clerk: getAuth() was called but Clerk can't detect usage of authMiddleware(). Please ensure the following:
- authMiddleware() is used in your Next.js Middleware.
- Your Middleware matcher is configured to match this route or page
```

After adding it to public routes all `publicProcedure` routes show up even when not signed in which is the intended behaviour.